samples/apps/smallbank/clients/small_bank_client.cpp

```diff
@@ -221,16 +221,16 @@ class SmallBankClient : public Base
       json j;
       j["name"] = to_string(account_it->get<size_t>());
       const auto response = conn->call("SmallBank_balance", j);
+      const auto response_body = conn->unpack_body(response);
 
-      auto result_it = response.find("result");
-      if (result_it == response.end())
+      if (response.status != HTTP_STATUS_OK)
       {
-        throw std::runtime_error(
-          "No result in verification response: " + response.dump());
+        throw std::runtime_error(fmt::format(
```

Comment:
Does anyone else want a THROW_FMT() or is it just me?

Reply:
Maybe - we do repeat this pattern a lot. I'd be tempted to keep the `throw` outside of the macro so that flow control is clear:

```
throw FMT_ERROR(std::runtime_error, "expected {}, got {}", expected, actual);
```